app/src/main/kotlin/cz/covid19cz/erouska/worker/WorkerRepository.kt

```diff
@@ -0,0 +1,30 @@
+package cz.covid19cz.erouska.worker
+
+import android.content.Context
+import androidx.work.*
+import cz.covid19cz.erouska.AppConfig
+import cz.covid19cz.erouska.exposurenotifications.worker.DownloadKeysWorker
+import java.util.concurrent.TimeUnit
+
+class WorkerRepository(private val context: Context) {
```

Comment:
This is related to exposure server, so I would move it to ExposureServerRepository